lib/shopify_api/rest/request.ex

```diff
@@ -61,8 +66,12 @@ defmodule ShopifyAPI.REST.Request do
     )
   end
 
+  def process_response_body(body) do
+    body |> Poison.decode()
+  end
+
   defp url(%{shop_name: domain}, path) do
-    "#{@transport}#{domain}/admin/#{path}"
+    "#{@transport}#{domain}/admin/api/#{version()}/#{path}"
```

Comment:
Interesting that they are only adding `/api` now

Reply:
yea! 💯 I had to triple check that to make sure I was using the correct path